citesphere/src/main/webapp/WEB-INF/tiles-defs.xml

```diff
@@ -51,5 +52,9 @@
      <definition name="auth/authority" extends="base.definition">
         <put-attribute name="content" value="/WEB-INF/views/auth/authorities/show.jsp" />    
     </definition>
+    <definition name="auth/authority/edit" extends="base.definition">
+        <put-attribute name="content" value="/WEB-INF/views/auth/authorities/editAuth.jsp" />    
```

Comment:
it's already in the folder 'authorities', so just call the file 'edit.jsp'

Reply:
okay